packages/devtools_app_shared/example/ui_example.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+
+class ExampleWidget extends StatelessWidget {
+  const ExampleWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // Shared component
```

Comment:
Maybe it's worth doing `import 'package:devtools_app_shared/ui.dart' as devtools;` to make it clear what widgets are from DevTools?